components/core/tools/scripts/lib_install/pipx-packages/install-cmake.sh

```
@@ -4,46 +4,32 @@ set -o errexit
-        echo "pipx failed to install the required version of CMake."
-        echo "To uninstall, run:"
-        echo "  pipx uninstall cmake"
-    fi
+    echo "pipx failed to install the required version of CMake."
+    echo "To uninstall, run:"
+    echo "  pipx uninstall cmake"
 
     exit 1
 fi
+
+echo "CMake version ${installed_version} installed at ${cmake_bin} satisfies version requirements."
```

Shellcheck (0.11.0)

[warning] cmake_bin is referenced but not assigned.

(SC2154)